Cargo.toml

```diff
@@ -39,6 +39,7 @@ version = "2"
 crossbeam = "0.2"
 futures = "0.1.9"
 futures-cpupool = "0.1.2"
+error-chain = "0.8.1"
```

Comment:
I'm afraid I'm guilty for that bad habit, but I think we should probably avoid putting minor in there. So 0.8 should be fine. I should have done it for futures, and futures-cpupool as well